app/models/force/application.rb

```diff
@@ -35,8 +38,10 @@ class Application < Force::ObjectBase
       { custom_api: 'householdVouchersSubsidies', domain: 'housing_voucher_or_subsidy', salesforce: 'Housing_Voucher_or_Subsidy' },
       { custom_api: 'housingCounselingAgency', domain: '', salesforce: 'Which_One' },
       { custom_api: 'id', domain: 'id', salesforce: 'Id' },
+      { custom_api: 'isFirstTimeHomebuyer', domain: 'is_first_time_homebuyer', salesforce: '?' },
```

Comment:
I left ? as the value for Salesforce in a lot of cases, focusing on domain to customAPI in this PR and just trying not to break Salesforce to domain translation that was already happening.